tools/test.py

```diff
@@ -471,6 +474,10 @@ def UnexpectedOutput(self):
       outcome = PASS
     return not outcome in self.test.outcomes
 
+  def HasSkipped(self):
+    s = '1..0 # Skipped:'
```

Comment:
This should probably be a regex if there's any concern about actual TAP compliance. The `Skipped` only needs to match `skip`, and it shouldn't be case sensitive.